recipes/watchdog/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "watchdog" %}
+{% set version = "0.8.3" %}
+{% set hash_type = "md5" %}
+{% set hash = "bb16926bccc98eae2a04535e4512ddf1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/w/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  entry_points:
+    - watchmedo = watchdog.watchmedo:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - argh >=0.24.1
+    - pathtools >=0.1.1
+    - pyyaml >=3.10
+    - setuptools
+  run:
+    - python
+    - argh >=0.24.1
+    - pathtools >=0.1.1
```

Comment:
Done in PRs ( https://github.com/conda-forge/staged-recipes/pull/1325 ) and ( https://github.com/conda-forge/staged-recipes/pull/1326 ) respectively.